src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: $transition-card-duration;
```

Comment:
Same

Reply:
Here I have both transform and color. Should I write transition: transform $transition-duration; ? As when I write both transform and color - it doesn't work